<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1607</width>
    <height>997</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="lw_ag_jobs_list">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>410</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>40</y>
      <width>411</width>
      <height>781</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/usecase_phone/usecase_phone.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>561</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="le_pickup">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>411</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Inffeldgasse 16a, 8010 Graz</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>pick up location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_deliver_at">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>411</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Herrengasse 10, 8010 Graz</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>delivery location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_description">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>411</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1x 1.5ECTS with grade ANY</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>description</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_post_job">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Job</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:600;&quot;&gt;Silvia's Monitor&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>170</y>
      <width>171</width>
      <height>147</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_ag_user">
       <property name="text">
        <string>agent@example.com</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_ag_passwd">
       <property name="text">
        <string>secret</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ag_login">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1069</x>
      <y>850</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>longitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="l">
    <property name="geometry">
     <rect>
      <x>1069</x>
      <y>880</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>latitude</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ag_long">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>850</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ag_lat">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>880</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ag_query_jobs">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>370</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Jobs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>820</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'u2400'; font-size:96px; color:#111111; background-color:#ffffff;&quot;&gt;➞&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ag_take">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>540</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Job</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lw_cust_jobs">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>421</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>My Jobs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>380</y>
      <width>61</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'u2400'; font-size:48pt; color:#111111; background-color:#ffffff;&quot;&gt;➜&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>360</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Details</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ag_radius">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>360</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>340</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>radius</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ag_resp">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>700</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;Reponse Code&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ag_update_pos">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>610</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Position</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_active_job">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>230</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;active job&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ag_fin_job">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>650</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Finish Job</string>
    </property>
   </widget>
   <widget class="QLabel" name="zzzz">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>881</width>
      <height>801</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/usecase_monitor/usecase_screen.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cust_refresh_jobs">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cust_details_jobs">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>490</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Details</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>-10</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:600;&quot;&gt;Michaels's Phone&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cust_resp">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>610</y>
      <width>441</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;Reponse Code&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_geo_translate">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>860</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Translate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_addr_query">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>860</y>
      <width>461</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Inffeldgasse 16a, 8010 Graz</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_token_customer">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>890</y>
      <width>721</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;token customer&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_token_agent">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>910</y>
      <width>721</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;token agent&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_job_details">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>380</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>810</y>
      <width>561</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_cust_user">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>admin@example.com</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_cust_passwd">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>secret</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cust_login">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>btn_cust_login</zorder>
    <zorder>le_cust_passwd</zorder>
    <zorder>label_2</zorder>
    <zorder>le_cust_user</zorder>
    <zorder>label</zorder>
   </widget>
   <zorder>zzzz</zorder>
   <zorder>label_3</zorder>
   <zorder>lw_ag_jobs_list</zorder>
   <zorder>frame</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_13</zorder>
   <zorder>l</zorder>
   <zorder>le_ag_long</zorder>
   <zorder>le_ag_lat</zorder>
   <zorder>btn_ag_query_jobs</zorder>
   <zorder>label_16</zorder>
   <zorder>btn_ag_take</zorder>
   <zorder>lw_cust_jobs</zorder>
   <zorder>label_17</zorder>
   <zorder>label_18</zorder>
   <zorder>label_19</zorder>
   <zorder>le_ag_radius</zorder>
   <zorder>label_20</zorder>
   <zorder>label_8</zorder>
   <zorder>lbl_ag_resp</zorder>
   <zorder>btn_ag_update_pos</zorder>
   <zorder>lbl_active_job</zorder>
   <zorder>btn_ag_fin_job</zorder>
   <zorder>btn_cust_refresh_jobs</zorder>
   <zorder>btn_cust_details_jobs</zorder>
   <zorder>label_12</zorder>
   <zorder>lbl_cust_resp</zorder>
   <zorder>btn_geo_translate</zorder>
   <zorder>le_addr_query</zorder>
   <zorder>lbl_token_customer</zorder>
   <zorder>lbl_token_agent</zorder>
   <zorder>te_job_details</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1607</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="usecase_phone.qrc"/>
 </resources>
 <connections/>
</ui>
